src/firecracker/src/main.rs

```diff
@@ -201,6 +198,11 @@ fn main_exitable() -> FcExitCode {
                     "Whether or not to include the file path and line number of the log's origin.",
                 ),
         )
+        .arg(
+            Argument::new("new-format")
+                .takes_value(false)
+                .help("Whether to use the new logging output format."),
+        )
```

Comment:
You seem to add this in the first commit and remove it later. Why?

Reply:
The 1st commit supported the old logger format and this format could be changed with this argument. The later commit removes the old format as in discussion we saw no reason to maintain the old format.